packages/@okta/vuepress-site/docs/guides/enable-express-configuration/main/index.md

```diff
@@ -0,0 +1,300 @@
+
+```bash
+auth0 actions create \
+  --name "express_configure_postlogin_action" \
+  --trigger "post-login" \
+  -r "node22" \
+  -s "SERVICE_INIT_LOGIN_URL='$SERVICE_INIT_LOGIN_URL'" \
+  -s "EXPRESS_CONFIGURE_API_IDENTIFIER=https://system.okta.com/actions/express-configure/v1" \
+  -s "TENANT_DOMAIN=https://$AUTH0_DOMAIN" \
+  --code "$(cat ./add_post_login_action.js)"
+```
+
+### Deploy the action
+
+Run the following command to deploy the action. From the prompt, select the action for the Express Configuration that was created in the previous step.
+
+``` bash
+auth0 actions deploy
+```
+
+### Attach the action to a flow
+
+Attach the action to a flow so that it executes as part of your tenant's traffic. See [Auth0 Actions](https://auth0.com/docs/customize/actions) to add action using the Auth0 dashboard.
+
+```bash
+auth0 api patch \
+  actions/triggers/post-login/bindings \
+  --data '{
+    "bindings": [
+      {
+        "ref": {
+          "type": "action_name",
+          "value": "express_configure_postlogin_action"
+        },
+        "display_name": "express_configure_postlogin_action"
+      }
+    ]
+  }'
+```
+
+## Step 6: Update tenant settings
+
+Update tenant settings to display the scope details on the consent page. This improves the user experience by providing information about the permissions being granted. Use the `use_scope_descriptions_for_consent` parameter to ensure that scope descriptions are shown instead of raw scope names.
+
+```bash
+auth0 api patch tenants/settings \
+  --data '{
+    "flags": {
+      "use_scope_descriptions_for_consent": true
+    }
+  }'
+```
+
+## Step 7: Email the Okta Express Configuration team
+
+Email following information to the Okta Express Configuration team at [expressconfig@okta.com](mailto:expressconfig@okta.com):
+
+1. Confirmation that you completed all the steps in this guide and your app is ready to support Express Configuration.
+2. Your app name in the OIN.
+3. The OIN Integration Client ID.
+
+The Okta Express Configuration team configures your app in the OIN and assigns it to your developer org. You can then test the feature by creating an instance of your app in the OIN catalog.
+
+# Verification and testing
+
+Follow these steps to verify and test the Express Configuration feature:
+
+1. Sign in to your [Okta Developer Edition org](/login/) as a user with either the super admin (`SUPER_ADMIN`) role, or the app (`APP_ADMIN`) and org (`ORG_ADMIN`) admin [roles](https://developer.okta.com/docs/api/openapi/okta-management/guides/roles/#standard-roles).
+1. Go to **Applications** > **Your OIN Integrations** in the Admin Console.
+1. Click **Browse App Catalog** and search for your app.
+1. Open your app's detail page and click **Add Integration**.
+1. In **General Settings**, click **Done** to create an instance of your OIN app.
+1. Go to the **Authentication** tab.
+1. Click **Configure SSO with OIDC** and complete the authorization flow.
```

Comment:
`complete the authorization flow.`

From user perspective, click the button will trigger authorize flow after which it'll perform Express Configure.